apps/sonda/docker-compose.yml

```diff
@@ -25,6 +26,7 @@ x-sonda-env: &sonda_env
 services:
   nwaku:
     image: ${NWAKU_IMAGE:-harbor.status.im/wakuorg/nwaku:v0.30.1}
+    container_name: nwaku
```

Comment:
( comment related to the previous line )
I think we can use a generic tag in the previous line so that the nwaku node version is updated automatically. For example, we could use: `harbor.status.im/wakuorg/nwaku:deploy-status-prod`